ONSdigital/response-operations-ui
# Removing next from session on sign out

At the moment, the session keeps a key that takes you to a page stored in the session once you signed in. This pull request removes it on sign out.

To test:

- Before logging in, go to /messages/bricks and log in

- Log back out

- When you log back in, it should take you to the homepage instead of /messages/bricks
